Game/LambdaHack/Server/Commandline.hs

```diff
@@ -9,162 +12,277 @@ import Game.LambdaHack.Common.Prelude
+
+automateAllP :: Parser Bool
+automateAllP =
+  switch (  long "automateAll"
+         <> help "give control of all UI teams to computer" )
+
+keepAutomatedP :: Parser Bool
+keepAutomatedP =
+  switch (  long "keepAutomated"
+         <> help "keep factions automated after game over" )
+
+newGameP :: Parser (Maybe Int)
+newGameP =
+  optional $ option auto
+    ( long "newGame"
+   <> help "start a new game, overwriting the save file, with difficulty for all UI players set to this value"
+   <> metavar "INT" )
+
+stopAfterSecsP :: Parser (Maybe Int)
+stopAfterSecsP =
+  optional $ option auto
+    ( long "stopAfterSeconds"
+   <> help "exit game session after around n seconds"
+   <> metavar "INT" )
+
+stopAfterFramesP :: Parser (Maybe Int)
+stopAfterFramesP =
+  optional $ option auto
+    ( long "stopAfterFrames"
+   <> help "exit game session after around n frames"
+   <> metavar "INT" )
+
+benchmarkP :: Parser Bool
+benchmarkP =
+  switch (  long "benchmark"
+         <> help "restrict file IO, print stats" )
+
+setDungeonRngP :: Parser (Maybe R.StdGen)
+setDungeonRngP = optional $
+  option auto (  long "setDungeonRng"
+              <> metavar "DUNGEON_RNG"
+              <> help "set dungeon generation RNG seed to string s" )
+
+setMainRngP :: Parser (Maybe R.StdGen)
+setMainRngP = optional $
+  option auto (  long "setMainRng"
+              <> metavar "MAIN_RNG"
+              <> help "set the main game RNG seed to string s" )
+
+dumpInitRngsP :: Parser Bool
+dumpInitRngsP =
+  switch (  long "dumpInitRngs"
+         <> help "dump RNG states from the start of the game" )
+
+dbgMsgSerP :: Parser Bool
+dbgMsgSerP =
+  switch (  long "dbgMsgSer"
+         <> help "let the server emit its internal debug messages" )
+
+gtkFontFamilyP :: Parser (Maybe Text)
+gtkFontFamilyP = optional $ T.pack <$>
+  strOption (  long "gtkFontFamily"
+            <> metavar "FONT_FAMILY"
+            <> help "use the given font family for the main game window in GTK" )
+
+sdlFontFileP :: Parser (Maybe Text)
+sdlFontFileP = optional $ T.pack <$>
+  strOption (  long "gtkFontFamily"
+            <> metavar "FONT_FILE"
+            <> help "use the given font file for the main game window in SDL2" )
+
+sdlTtfSizeAddP :: Parser (Maybe Int)
+sdlTtfSizeAddP = optional $
+  option auto (  long "sdlTtfSizeAdd"
+              <> metavar "INT"
+              <> help "enlarge map cells over scalable font max height in SDL2" )
+
+sdlFonSizeAddP :: Parser (Maybe Int)
+sdlFonSizeAddP = optional $
+  option auto (  long "sdlFonSizeAdd"
+              <> metavar "INT"
+              <> help "enlarge map cells on top of .fon font max height in SDL2" )
+
+fontSizeP :: Parser (Maybe Int)
+fontSizeP = optional $
+  option auto (  long "fontSize"
+              <> metavar "INT"
+              <> help "use the given font size for the main game window" )
+
+noColorIsBoldP :: Parser (Maybe Bool)
+noColorIsBoldP = flag Nothing (Just False)
+                  (  long "noColorIsBold"
```

Comment:
Indentation? One more space? Dollar in the `flag` line instead of the parens and move this line to column 2? Move the `flag` line to column 2? I bet there are other sensible options as well.